tests/testthat/test-nodeCentricSteinerTreeProblem.R

```diff
@@ -196,9 +185,22 @@ test_that("Trying MWCS solver against the lymphoma.stp instance in the ACTMOD St
 
   skip("SteinLib drosophila test takes too long even using CPLEX - although it does pass!")
 
-  drosophila001_igraph <- readMWCSgraph('./testData/ACTMOD/drosophila001.stp')
+  #drosophila001_igraph <- readMWCSgraph('./testData/ACTMOD/drosophila001.stp')
 
   # Takes around five minutes with CPLEX
-  #drosophila001_MWCS <- solveNodeCentricSteinerTreeProblem(drosophila001_igraph, timeOut = 7200,verbosity = TRUE, solverChoice = "RCPLEX")
+  #drosophila001_MWCS <- nodeCentricSteinerTreeProblem$new(drosophila001_igraph, verbose = T)$findSingleSteinerSolution(100)
+})
+
+
```

Comment:
I've added this test because I kept breaking this warning when I was trying to solve the issue with the solver time limit.

Reply:
Good idea.

Does it not add some time to the testing process? Perhaps lower solverTimeLimit and force the warning to occur? 60 seconds should be sufficient.